ui/components/component-library/button-primary/button-primary.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { ButtonBase } from '../button-base';
+import { BUTTON_SIZES } from './button-primary.constants';
+
+export const ButtonPrimary = ({ className, children, danger, ...props }) => {
+  return (
+    <ButtonBase
+      className={classnames(className, 'mm-button-primary', {
+        'mm-button-primary--type-danger': Boolean(danger),
+      })}
+      {...props}
+    >
+      {children}
+    </ButtonBase>
+  );
+};
+
+ButtonPrimary.propTypes = {
+  /**
+   * The size of the ButtonPrimary.
+   * Possible values could be 'BUTTON_SIZES.SM', 'BUTTON_SIZES.MD', 'BUTTON_SIZES.LG',
+   * Default value is 'BUTTON_SIZES.MD'.
```

Comment:
Should we be using the general size object reference here?

```suggestion
   * Possible values could be 'SIZES.SM', 'SIZES.MD', SIZES.LG',
   * Default value is 'SIZES.MD'.
```